src/Common/Commands.Common.Graph.RBAC/ActiveDirectory/ActiveDirectoryClientExtensions.cs

```diff
@@ -59,7 +59,8 @@ public static PSADObject ToPSADObject(this AADObject obj)
                     DisplayName = obj.DisplayName,
                     Type = obj.ObjectType,
                     Id = new Guid(obj.ObjectId),
-                    SecurityEnabled = obj.SecurityEnabled
+                    SecurityEnabled = obj.SecurityEnabled,
+                    MailNickname = obj.Mail
```

Comment:
>MailNickname = obj.Mail [](start = 20, length = 23)

for a group the mail will be always null,do we need this?
[https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/groups-operations](https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/groups-operations)